ibis/omniscidb/compiler.py

```diff
@@ -180,6 +180,7 @@ class OmniSciDBTableSetFormatter(compiles.TableSetFormatter):
     _join_names = {
         ops.InnerJoin: 'JOIN',
         ops.LeftJoin: 'LEFT JOIN',
+        ops.LeftSemiJoin: 'JOIN',  # needed by topk as filter
```

Comment:
not sure if it is the best approach ... but at least it is working now.
maybe #512 would be a better approach to fix this issue

Reply:
that's a really old issue, has things changed in the interim?

Reply:
it seems it was not changed. but maybe we need to check this better.